service/history/workflowTaskHandler.go

```diff
@@ -54,12 +54,13 @@ type (
 		namespaceEntry          *cache.NamespaceCacheEntry
 
 		// internal state
-		hasBufferedEvents           bool
-		failWorkflowTaskInfo        *failWorkflowTaskInfo
-		activityNotStartedCancelled bool
-		continueAsNewBuilder        mutableState
-		stopProcessing              bool // should stop processing any more commands
-		mutableState                mutableState
+		hasBufferedEvents                 bool
+		failWorkflowTaskInfo              *failWorkflowTaskInfo
+		activityNotStartedCancelled       bool
+		continueAsNewBuilder              mutableState
+		stopProcessing                    bool // should stop processing any more commands
+		mutableState                      mutableState
+		initiatedChildExecutionsInSession map[string]struct{} // Set of initiated child executions in the workflow task
```

Comment:
session is a new term. Should we call more like currentCommandBatch or something else?

Reply:
Changed to `initiatedChildExecutionsInBatch`